examples/text_classification/README.md

```diff
@@ -0,0 +1,41 @@
+# Amazon review sentiment analysis
```

Comment:
Add a model's goal first. something like "The goal of this model to classify sentiments based on _such_data_ by using bla bla bla and bla bla"